src/test/java/games/strategy/engine/lobby/server/ModeratorControllerTest.java

```diff
@@ -16,6 +16,8 @@
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
 
+import games.strategy.engine.lobby.server.userDB.DbUser;
+import games.strategy.engine.lobby.server.userDB.HashedPassword;
 import games.strategy.engine.lobby.server.userDB.DbUserController;
```

Comment:
Checkstyle: "Wrong lexicographical order for 'games.strategy.engine.lobby.server.userDB.DbUserController' import. Should be before 'games.strategy.engine.lobby.server.userDB.HashedPassword'."